### PR TITLE
make it work with latest solang

### PR DESCRIPTION
**Pull request by gelfand**

### CODE REVIEW

File: ethers-solc/src/resolver.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ use std::{
 
 use rayon::prelude::*;
 use semver::VersionReq;
-use solang_parser::pt::{Import, SourceUnitPart};
```
Comment (mattsse) [inline ethers-solc/src/resolver.rs:36]: don't think that's needed

File: ethers-solc/src/resolver.rs [hunk 2]
```diff
@@ -282,7 +282,7 @@ impl Graph {
                     utils::source_name(&self.nodes[idx].path, &self.root).display(),
                     utils::source_name(&self.nodes[dep].path, &self.root).display()
                 );
-                continue
+                continue;
```
Comment (mattsse) [inline ethers-solc/src/resolver.rs:285]: please use nightly formatting `cargo +nightly fmt`

**Review by mattsse — CHANGES_REQUESTED** [review]
thanks,
two nits :)

File: ethers-solc/src/resolver.rs (hunk 2 above)
Comment (gelfand) [reply ethers-solc/src/resolver.rs:285]: my bad (

**Review by gakonst — APPROVED** [review]

**Pull request merged**